sql/updates/world/master/yyyy_mm_dd_xx_world_temple_of_kotmogu.sql

```diff
@@ -0,0 +1,277 @@
+SET @CGUID := 20000000000;
+SET @OGUID := 20000000000;
+SET @SPAWNGROUP := 200000;
+SET @ATPROPERTIESID := 200000000; -- CHANGE ME IN SCRIPT
```

Comment:
My recommendation would be using the exact same entry for both Id and AreaTriggerId in case we lack Properties, since we got IsCustom, there shouldn't be any issues.